.github/disabled/e2e-angular-cli-workflow.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 #          ng build --aot
   npm:
     name: 'npm:angular example'
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
Isn't the matrix setup still missing here?

Reply:
Those are disabled, I moved there to don't block development, still need few changes to make then run with the new setup.

Reply:
Ah right. But normally I would prefer to add the matrix with such changes but it's ok for now =)